6001-7000/LIT6018MalkaSanbat.xml

```diff
@@ -0,0 +1,117 @@
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="GundaGunde144"></witness>
+                    <witness corresp="BAVet276"></witness>
+                    <witness corresp="EMML6993"></witness>
+                    <witness corresp="EAP526112"></witness>
+                </listWit>
+                
+                
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change when="2024-08-14" who="ES">created record as suggested in issue https://github.com/BetaMasaheft/Documentation/issues/2604 by Augustine Dickinson</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listRelation><relation name="saws:isDifferentTo" active="LIT6018MalkaSanbat" passive="LIT2889RepCh170"></relation>
+                <relation name="saws:isDifferentTo" active="LIT6018MalkaSanbat" passive="LIT6179MalkaSanbat"></relation></listRelation>
+            <div type="edition" xml:lang="gez">
+                <div type="textpart">
+                    <lg type="stanza" n="1">
+                        <l n="1">ድርሳን፡ ዘደረሰ፡ አሐዱ፡ ርቱዐ፡ ሃይማኖት፨</l>
+                        <l n="2">ሶበ፡ አገበሮ፡ ኪያሁ፡ ኅሊና፡ ፍትወት፨</l>
+                        <l n="3">በኵሉ፡ እሑድ፡ ወበሰናብት፨</l>
+                        <l n="4">ይትነበብ፡ ይቤ፡ መጽሐፈ፡ ዝንቱ፡ ማሕሌት፨</l>
+                        <l n="5">ዘኍላቌሁ፡ ሰኑይ፡ ወኃምስ፡ አርእስት፨</l>
+                    </lg>
+                    
+                    <lg type="stanza" n="2">
+                        <l n="1">ሶበ፡ አእመርኩ፡ በስምኪ፡ ከመ፡ ታቦተ፡ ይቄድሱ፨</l>
+                        <l n="2">ወድሶተኪ፡ ወጠንኩ፡ ለአእምሮትየ፡ መጠነ፡ ናእሱ፨</l>
+                        <l n="3">ሰንበተ፡ ክርስቲያ፡ ቅድስት፡ ንግሥተ፡ መሳፍንት፡ ስሱ፨</l>
+                        <l n="4">እለሰ፡ ኪያኪ፡ አርኰሱ፨</l>
+                        <l n="5">ፈቀዱ፡ ሞተ፡ ወኵነኔ፡ ኀሠሡ፨</l>
+                    </lg>
+                    
+                    <lg type="stanza" n="3">
+                        <l n="1">ምስለ፡ ዕለታት፡ ኃምስ፡ እለ፡ እምቅድሜሁ፡ ተፈጥሩ፨</l>
+                        <l n="2">አመ፡ ዕለተ፡ ፍዳ፡ ወደይን፡ ለእግዚአብሔር፡ ዕለተ፡ ምሥጢሩ፨</l>
+                        <l n="3">እስመ፡ የኀልፍ፡ ጥቀ፡ ወይሠዐር፡ ዝክሩ፨</l>
+                        <l n="4">ለእስራኤል፡ እምነ፡ ሰንበቶሙ፡ ለሰንበትነ፡ ከመ፡ የዐቢ፡ ክብሩ፨</l>
+                        <l n="5">ዘኢያእመርክሙ፡ ሰብእ፡ በዝንቱ፡ አእምሩ፨</l>
+                    </lg>
+                    
```

Comment:
@DenisNosnitsin1970 Perhaps this additional selection of stanzas also would help to clarify the text's relation to the genre? I think it seems clear that it loosely maps onto the usual format, with ሀልዎት፡ standing in for ፅንሰት፡, ተከሥቶት፡ for ልደት፡, and ዝክር፡ for ዝክረ፡ ስም፡.
```suggestion
                    <lg type="stanza" n="6">
                      <l n="1">ሰላም፡ ለሀልዎትኪ፡ እምቅድመ፡ ኦርያሬስ፡ ወብርሃን፨</l>
                      <l n="2">ሰንበተ፡ ክርስቲያን፡ ቅድስት፡ ንግሥተ፡ አዝማናት፡ ኵሎን፨</l>
                      <l n="3">ኪያኪሰ፡ እለ፡ ይሜንኑ፡ አብዳን፨</l>
                      <l n="4">ኢይትባልሑ፡ እምወይሌ፡ ወኢያመሥጡ፡ እምደይን፨</l>
                      <l n="5">እመ፡ ቆሙ፡ ኖኅ፡ ወሙሴ፡ መስፍን፨</l>
                    </lg>
                    <lg type="stanza" n="7">
                      <l n="1">ሰላም፡ ለተከሥቶትኪ፡ በአትልዎ፡ ዘመድ፡ ሰብዑ፨</l>
                      <l n="2">እማሕፀነ፡ ወለት፡ ወእም፡ እምቅድመ፡ ፍጥረታት፡ ይፃኡ፨</l>
                      <l n="3">ሰንበተ፡ ክርስቲያን፡ ቅድስት፡ ለእግዚአብሔር፡ ዕለተ፡ ስምዑ፨</l>
                      <l n="4">ስብሐተ፡ ዚአኪ፡ አመ፡ ውስተ፡ ዓለም፡ መጽኡ፨</l>
                      <l n="5">ነቢያት፡ ዜነዉ፡ ወአርድእት፡ አይድዑ፨</l>
                    </lg>
                    <lg type="stanza" n="8">
                      <l n="1">ሰላም፡ ለዝክርኪ፡ ውስተ፡ አፈ፡ ክሉ፡ ዘተሰብሐ፨</l>
                      <l n="2">ጎርጎሬዎስ፡ ልዑለ፡ ዜና፡ ከመ፡ ነገረ፡ በሮሃ፨</l>
                      <l n="3">ሰንበተ፡ ክርስቲያን፡ ቅድስት፡ ለሰንበተ፡ አይሁድ፡ መናቅዕቲሃ፨</l>
                      <l n="4">ወለንጊኖስሂ፡ ኀደገ፡ ሶቤሃ፨</l>
                      <l n="5">ሞገሰ፡ ዚአኪ፡ ጽሒፎ፡ ብዙኀ፨</l>
                    </lg>
```

Reply:
Explanation in abstract is ok, and additional stanzas are also helpful (still a kind a descriptive added title would facilitate the orientation, as the first signal to the user).